flashrom.c

```diff
@@ -1752,6 +1796,13 @@ int prepare_flash_access(struct flashctx *const flash,
 		return 1;
 	}
 
+	if ((write_it || erase_it) && !flash->flags.force) {
+		if(write_protect_check(flash)) {
```

Comment:
```suggestion
		if (write_protect_check(flash)) {
```

Please make code style consistent

Reply:
[c80f032](https://github.com/Dasharo/flashrom/pull/17/commits/c80f0327d3a26e14236019c926902d01267135ee)